crates/bevy_sprite/src/lib.rs

```diff
@@ -69,6 +71,22 @@ pub enum SpriteSystem {
     ComputeSlices,
 }
 
+/// A component that marks entities that aren't themselves sprites but become
+/// sprites during rendering.
+///
+/// Right now, this is used for `Text`.
+#[derive(Component, Reflect, Clone, Copy, Debug, Default)]
+#[reflect(Component, Default)]
+pub struct SpriteSource;
```

Comment:
I don't quite understand why we need this, but I also couldn't get this working with just `Text` so I assume it's necessary.

Reply:
Hmm, actually I think I came up with a way to make that no longer necessary. I'll try it tonight.

Reply:
Actually, never mind, it'd be uglier than the current solution.

The problem is that `Text` uses the sprite visibility checking system. But that system can't name `Text` as it's an upstream crate. So we need some marker component that the sprite visibility checker can name.

Reply:
BTW, the idea I had was to introduce a "label" type that's different from the component type. That way `Sprite` and `Text` would have the same label. But then you'd need two `check_visibility` systems, one for each component, that together populate the *same* `VisibleEntities` list. I think that's uglier than just using `SpriteSource`.